### ISSUE TITLE
Something wrong with code block

### ISSUE THREAD

**Issue by ubo5** [typora/typora-issues#832]
When type **quote** in **code block**, the Typora(Version 0.9.9.10.2 (0.9.9.10.2)) will be  unresponsive. By the way my sys is  MacOS Sierra 10.12.16.

**Comment by shoesforindustry** [comment]
This will crash it on my OSX, it was what was causing my problem opening files by double clicking - because this was in the file
```php
    $link.=' 
```

**Comment by abnerlee** [comment]
We will fix this soon

**Comment by solarce** [comment]
I think I am running into this, it's triggered for me when I add a language to a code block

This is fine
```
hi 
```

But if I do
``` ruby
hi
```

The app locks up and I have to force quite.

I'm using Typora `version 0.9.9.10.2 (0.9.9.10.2)` and macOS:
```
ProductName:    Mac OS X
ProductVersion: 10.11.6
BuildVersion:   15G1217
```

**Comment by abnerlee** [comment]
Fixed in 0.9.9.10.3
